test_merge.py

```diff
@@ -34,6 +34,26 @@ def test_extra_key(self):
         output = merge(raw, master)
         self.assertEqual(gold, output)
 
+    def test_nested_master_has_more_default(self):
+        raw = {
+            "foo": {
+                "bar": 1
+            }
+        }
+        master = {
+            "foo": {
+                "bar": None,
+                "baz": None
+            }
+        }
+        gold = {
+            "foo": {
+                "bar": 1
+            }
+        }
+        output = merge(raw, master)
+        self.assertEqual(gold, output)
```

Comment:
Why do you use gold instead of expected?

Reply:
not really sure lol I think the convention of saying "this is the golden value I want to see" for some reason just stuck when I was figuring out how to write better tests a long time ago. "expected" would make me just as happy, not really partial